libs/sdk-ui-kit/src/DateDatasets/dateDatasets.ts

```diff
@@ -0,0 +1,160 @@
+// (C) 2007-2021 GoodData Corporation
+import groupBy from "lodash/groupBy";
+import takeWhile from "lodash/takeWhile";
+import first from "lodash/first";
+
+import { IDateDataset, IDateDatasetHeader } from "./typings";
+
+/**
+ * @internal
+ */
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const isDateDatasetHeader = (obj: any): obj is IDateDatasetHeader => {
```

Comment:
Please use `obj: unknown` for typeguard parameters. This will allow you to get rid of the eslint disabling comment.

Reply:
Fixed.